trl/commands/sft.py

```diff
@@ -0,0 +1,148 @@
+# flake8: noqa
```

Comment:
So i guess this means we can't use `trl/examples/scripts/sft.py` directly for this?

Reply:
yeah for now but I think we can move the example folders there, let me think a bit